go/vt/vttablet/tabletserver/throttle/throttler.go

```diff
@@ -479,6 +483,10 @@ func (throttler *Throttler) Open() error {
 					throttler.initMutex.Lock()
 					defer throttler.initMutex.Unlock()
 					throttler.applyThrottlerConfig(ctx, throttlerConfig) // may issue an Enable
+					go throttler.watchSrvKeyspaceOnce.Do(func() {
+						// We start watching SrvKeyspace only after we know it's been created. Now is that time!
+						throttler.srvTopoServer.WatchSrvKeyspace(context.Background(), throttler.cell, throttler.keyspace, throttler.WatchSrvKeyspaceCallback)
+					})
```

Comment:
I guess the other fix to only watch the SrvKeyspace record once it exists was cherry-picked into this PR? But not the one that removed `--enable-throttler`.

Reply:
That's correct